### PR TITLE
New sim export

### PR DESCRIPTION
**Pull request by Jyborn**
run simulation now returns vcss, designs and vds used.

Thinks that might be wrong not sure:
 - surplus_value_end_result is just the last point of the simulation.
 - payback_time is the same as mean_payback_time 
  mean_npv_payback_time is described as: 
        Calculates the time it takes before the mean npv 
        starts to generate value. If the mean NPV doesn't
        go above 0 then -1 is returned. 
Sounds to me like it is the same thing as payback_time should be

### CODE REVIEW

File: sedbackend/apps/cvs/simulation/router.py [hunk 1]
```diff
@@ -43,15 +44,15 @@ async def run_dsm_file_simulation(native_project_id: int, sim_params: models.Fil
 @router.post(
     '/project/{native_project_id}/simulation/run-multiprocessing',
     summary='Run monte carlo simulation with multiprocessing',
-    response_model=List[models.Simulation],
+    response_model=models.SimulationResult,
     dependencies=[Depends(SubProjectAccessChecker(AccessLevel.list_can_read, CVS_APP_SID))]
 )
 async def run_multiprocessing(sim_settings: models.EditSimSettings, vcs_ids: List[int],
-                              design_group_ids: List[int],
+                              design_group_ids: List[int],  native_project_id: int,
```
Comment (obennet) [inline sedbackend/apps/cvs/simulation/router.py:51]: Add parameters in order of "hierarchy". Project -> vcs -> design group etc

File: sedbackend/apps/cvs/simulation/router.py [hunk 2]
```diff
@@ -43,15 +44,15 @@ async def run_dsm_file_simulation(native_project_id: int, sim_params: models.Fil
...
                               normalized_npv: Optional[bool] = False,
-                              user: User = Depends(get_current_active_user)) -> List[models.Simulation]:
-    return implementation.run_simulation(sim_settings, vcs_ids, design_group_ids, user.id, normalized_npv,
-                                         True)
+                              user: User = Depends(get_current_active_user)) -> SimulationResult:
+    return implementation.run_simulation(sim_settings, vcs_ids, design_group_ids, user.id, native_project_id,
```
Comment (obennet) [inline sedbackend/apps/cvs/simulation/router.py:54]: hierarchy order

File: sedbackend/apps/cvs/simulation/storage.py [hunk 3]
```diff
@@ -88,14 +90,14 @@ def get_dsm_from_file(db_connection: PooledMySQLConnection, user_id: int, projec
 
 
 def run_simulation(db_connection: PooledMySQLConnection, sim_settings: models.EditSimSettings,
-                   vcs_ids: List[int],
+                   vcs_ids: List[int], project_id: int,
```
Comment (obennet) [inline sedbackend/apps/cvs/simulation/storage.py:93]: hierarchy order

**Review by obennet — CHANGES_REQUESTED** [review]
Looks good except for the comments

**Comment by Jyborn** [comment]
Hierarchy of params should be fixed could not find any in the wrong order

**Pull request merged**